### PR TITLE
Rent

### PR DESCRIPTION
**Pull request by lucasenriquez**
## Summary

- [ ] I have created the Rent model and test
- [ ] BEFORE MERGE: I have updated the TRELLO card column before merging the pull request

## TRELLO Card

https://trello.com/c/AdtbGRUM/42-crear-tests-para-el-modelo-de-rent-y-user

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+  validates :user, presence: true
+  validates :book, presence: true
```
Comment (melissaregnier) [inline app/models/rent.rb:5]: Consider adding a validation to the :from and :to fields

File: db/migrate/20210315184216_create_rents.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  def up
+    create_table :rents do |t|
+      t.references :user, foreign_key: true
+      t.references :book, foreign_key: true
+      t.datetime :from
+      t.datetime :to
+
+      t.timestamps
+    end
+  end
```
Comment (melissaregnier) [inline db/migrate/20210315184216_create_rents.rb:11]: Add a newline between these two methods

File: spec/factories/rents.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :rent do
+    user { create(:user) }
+    book { create(:book) }
+    from { "2021-03-15 18:42:17" }
```
Comment (melissaregnier) [inline spec/factories/rents.rb]: You can use the Faker gem here

File: spec/models/rent_spec.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+describe Rent, type: :model do
+  it do
+    should validate_presence_of(:user)
+  end
+
+  it do
+    should validate_presence_of(:book)
+  end
+
+  subject(:rent) do
```
Comment (melissaregnier) [inline spec/models/rent_spec.rb]: Usually when it does not span more than one line, you can put this in one line like this:
`subject(:rent) { build(:rent) }`
This applies to `let` blocks too.

**Review by melissaregnier — CHANGES_REQUESTED** [review]

File: app/models/rent.rb (hunk 1 above)
Comment (lucasenriquez) [reply app/models/rent.rb:5]: I just did it

File: spec/factories/rents.rb (hunk 3 above)
Comment (lucasenriquez) [reply spec/factories/rents.rb]: Done

File: db/migrate/20210315184216_create_rents.rb (hunk 2 above)
Comment (lucasenriquez) [reply db/migrate/20210315184216_create_rents.rb:11]: Done

File: spec/models/rent_spec.rb (hunk 4 above)
Comment (lucasenriquez) [reply spec/models/rent_spec.rb]: Done, thanks!

**Review by melissaregnier — APPROVED** [review]

**Review by NicoFunes — APPROVED** [review]

**Pull request merged**